docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -158,4 +163,31 @@ Given I start mobile application with capabilities:
 |appium:options.clearSystemFiles|false|
 ----
 
+=== Tap on element
+
+==== *_Info_*
+
+Taps on element located by locator with specified duration
+
+==== *_Wording_*
+
+[source,gherkin]
+----
+When I tap on element located by `$locator` with duration `$duration`
+----
+
+==== *_Parameters_*
+
+. `$locator` - <<_locator>>
+. `$duration` - tap duration in https://en.wikipedia.org/wiki/ISO_8601#Durations[ISO 8601 format]
```

Comment:
I think it requires explanation for end-users what tap duration is

Reply:
We have a global attribute for this